java/src/jmri/jmrit/beantable/LRouteTableAction.java

```diff
@@ -970,7 +970,7 @@ public void actionPerformed(ActionEvent e) {
             tab2.add(new JLabel(rbx.getString("OutputTitle")));
             _outputAllButton = new JRadioButton(Bundle.getMessage("All"), true);
             JRadioButton includedOutputButton = new JRadioButton(Bundle.getMessage("Included"), false);
-            tab2.add(makeShowButtons(_outputAllButton, includedOutputButton, null, Bundle.getMessage("Show")));
+            tab2.add(makeShowButtons(_outputAllButton, includedOutputButton, null, "Show")); // NOI18N
```

Comment:
Thanks. String "Show" is passed to method makeShowButtons() that in turn looks up a property. I introduced the bug myself after misreading, so at this spot NO18N seems appropriate. Will look into improving a possible translation inside the method for in 4.5.1 but for now wanted to get the error out of 4.4.4.